platform/public_api_guide.md

```diff
@@ -0,0 +1,59 @@
+Public API guide
+========================
+
+This page describes how to access Piwik PRO API which uses 
+[client credentials](https://www.oauth.com/oauth2-servers/access-tokens/client-credentials/)
+Oauth grant type for obtaining user token. 
```

Comment:
We should probably use official notation - `OAuth` -> https://en.wikipedia.org/wiki/OAuth